crates/shared/src/price_estimation.rs

```diff
@@ -219,6 +187,30 @@ pub struct Arguments {
     /// can be paid out of the settlement contract buffers.
     #[clap(long, env, default_value = "1.")]
     pub quote_inaccuracy_limit: f64,
+
+    /// How strict quote verification should be.
+    #[clap(
+        long,
+        env,
+        default_value = "unverified",
+        value_enum,
+        verbatim_doc_comment
+    )]
+    pub quote_verification: QuoteVerificationMode,
+}
+
+/// Controls which level of quote verification gets applied.
+#[derive(Copy, Clone, Debug, clap::ValueEnum)]
+#[clap(rename_all = "kebab-case")]
+pub enum QuoteVerificationMode {
```

Comment:
pico-nano-nit
```suggestion
pub enum QuoteVerification {
```

If we had a quote module maybe even using this as `quote::Verification` might be slick?

Reply:
Currently there is no quote module that would nicely facilitate that.
I'm sure once I remove all the old price estimation logic I'll get around to cleaning some of these things up as well so I will leave this for now.